examples/gno.land/r/sys/users/verifier.gno

```diff
@@ -0,0 +1,36 @@
+package users
```

Comment:
This file is special because it allows gno.land to efficiently check for namespaces. 

We should unexport everything; from the gnosdk perspective, a function does not need to be exported to be callable. 

Additionally, the file should include more comments explaining its purpose, ideally linking to the vmkeeper file that uses it.

Reply:
Edit: We will move this to `r/sys/names`, which will be responsible for calling `r/sys/users` and `r/sys/teams` following this flow:

1. Is the name exactly the caller's address?
2. If not, does ResolveByName(name) match the caller's address?
3. If not, is the resolved user actually a team where the caller has the correct permissions?
4. If none of the above, return false.

The "enabled" and "checkFunc" updates should be in `r/sys/names`. `r/sys/users` can be paused without impacting `r/sys/names` and vice versa.

Reply:
Moved to `r/sys/names`, added tests, fixed other comments related to this.